libnavigator/src/main/java/com/mapbox/navigation/navigator/MapboxNativeNavigatorImpl.kt

```diff
@@ -257,11 +256,11 @@ object MapboxNativeNavigatorImpl : MapboxNativeNavigator {
         }
 
```

Comment:
For consistency, could we update https://github.com/mapbox/mapbox-navigation-android/blob/68345f96d58741df28324f71451f80c20b599523/libnavigator/src/main/java/com/mapbox/navigation/navigator/MapboxNativeNavigatorImpl.kt#L180